hathor/verification/fee_header_verifier.py

```diff
@@ -0,0 +1,58 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from __future__ import annotations
+
+from typing import Sequence
+
+from hathor.transaction.exceptions import FeeHeaderTokenNotFound, InvalidFeeHeader
+from hathor.transaction.headers import FeeHeader
+
+MAX_FEES_LEN: int = 16
+
+
+class FeeHeaderVerifier:
+
+    @staticmethod
+    def verify_without_storage(fee_header: 'FeeHeader', tx_tokens_len: int) -> None:
```

Comment:
We usually do not specify which verification will call each verifier. So I'd rename this method to `verify_fee_list()`.